ads/appnexus.js

```diff
@@ -24,103 +24,111 @@ const APPNEXUS_AST_URL = 'https://acdn.adnxs.com/ast/ast.js';
  * @param {!Object} data
  */
 export function appnexus(global, data) {
-  const args = [];
-  args.push('size=' + data.width + 'x' + data.height);
-  if (data.tagid) {
-    validateData(data, ['tagid']);
-    args.push('id=' + encodeURIComponent(data.tagid));
-    writeScript(global, constructTtj(args));
-    return;
-  } else if (data.member && data.code) {
-    validateData(data, ['member', 'code']);
-    args.push('member=' + encodeURIComponent(data.member));
-    args.push('inv_code=' + encodeURIComponent(data.code));
-    writeScript(global, constructTtj(args));
-    return;
-  }
-
-  /**
-   * Construct the TTJ URL. Note params should be properly encoded first (use
-   * encodeURIComponent);
-   * @param  {!Array<string>} args query string params to add to the base URL.
-   * @return {string}      Formated TTJ URL.
-   */
-  function constructTtj(args) {
-    let url = 'https://ib.adnxs.com/ttj?';
-    for (let i = 0; i < args.length; i++) {
-      //append arg to query. Please encode arg first.
-      url += args[i] + '&';
+    const args = [];
+    args.push('size=' + data.width + 'x' + data.height);
+    if (data.tagid) {
+        validateData(data, ['tagid']);
+        args.push('id=' + encodeURIComponent(data.tagid));
+        writeScript(global, constructTtj(args));
+        return;
+    } else if (data.member && data.code) {
+        validateData(data, ['member', 'code']);
+        args.push('member=' + encodeURIComponent(data.member));
+        args.push('inv_code=' + encodeURIComponent(data.code));
+        writeScript(global, constructTtj(args));
+        return;
     }
 
-    return url;
-  }
+    /**
+     * Construct the TTJ URL. Note params should be properly encoded first (use encodeURIComponent);
+     * @param  {!Array<string>} args query string params to add to the base URL.
+     * @return {string}      Formated TTJ URL.
+     */
+    function constructTtj(args) {
+        let url = 'https://ib.adnxs.com/ttj?';
+        for (let i = 0; i < args.length; i++) {
+            //append arg to query. Please encode arg first.
+            url += args[i] + '&';
+        }
+
+        return url;
+    }
 
-  appnexusAst(global, data);
+    appnexusAst(global, data);
 
 }
 
-/**
- * @param {!Window} global
- * @param {!Object} data
- */
 function appnexusAst(global, data) {
```

Comment:
add back removed jsdoc.

Reply:
oh, sure, just copied from 14038, will add it agan